week-9/Homework/mandatory/1-practice/2-code-reading.md

```diff
@@ -61,3 +65,6 @@ console.log(y);
 ```
 
 What will be the output of this code. Explain your answer in 50 words or less.
+
+Explanation 3 :
+Basically, We are doing the same thing. But the result was not affected by function at the first one in contrast to second one. Because we assign a number at the first, an object at the second. line 64 is 10: line 54 is 9.
```

Comment:
Yes, that's right. 
When we pass in a number we are `passing by value`. Changing the argument passed into the function will not change the variable outside of the function.

When we pass in an object, we are `passing by reference`. This means that if you make changes to the argument if will affect the object in the variable outside the function.

https://medium.com/nodesimplified/javascript-pass-by-value-and-pass-by-reference-in-javascript-fcf10305aa9c